lib/usdUfe/ufe/Utils.h

```diff
@@ -204,6 +210,14 @@ Ufe::Selection removeDescendants(const Ufe::Selection& src, const Ufe::Path& fil
 USDUFE_PUBLIC
 Ufe::Selection recreateDescendants(const Ufe::Selection& src, const Ufe::Path& filterPath);
 
+#ifdef UFE_SCENEITEM_HAS_METADATA
+//! Converts a UFE Value to a VtValue
+USDUFE_PUBLIC PXR_NS::VtValue ufeValueToVtValue(const Ufe::Value& ufeValue);
+
+//! Converts a VtValue to a UFE Value
+USDUFE_PUBLIC Ufe::Value vtValueToUfeValue(const PXR_NS::VtValue& vtValue);
+#endif
```

Comment:
I think these are useful functions in general and not just for the metadata stuff. I would ifdef them on Ufe v3 which is when value.h was added.

Reply:
Does V3 start at Maya 2023? 
The reason for wrapping it in this was that Maya 2022 compilations were failing due to line 33 #include <ufe/value.h>:

> Utils.h(30): fatal error C1083: Cannot open include file: 'ufe/value.h': No such file or directory

Reply:
Yes exactly. Maya 2022 contains Ufe v2. Maya 2023 contains v3. Maya 2024 contains v4.
Ufe::Value was added to Ufe v3.